tool/gravity/cli/config.go

```diff
@@ -157,9 +158,22 @@ type InstallConfig struct {
 	FromService bool
 	// AcceptEULA allows to auto-accept end-user license agreement.
 	AcceptEULA bool
+
+	// Computed values
+	//
 	// writeStateDir is the directory where installer stores state for the duration
 	// of the operation
 	writeStateDir string
+	// app is the application being installed
+	app *app.Application
```

Comment:
Moved to top-level installer configuration to enable early validation - otherwise, if the configuration is validated in the server component, the error is a cryptic:
```
service 'gravity-installer.service' has failed. Check journal log for details.
```